### PR TITLE
Getting PR Status from github

### PR DESCRIPTION
**Pull request by SekharV**
Getting PR Status from Github and displaying in UI

### CODE REVIEW

File: HacktoberfestProject.Web/Views/Home/Index.cshtml [hunk 1]
```diff
@@ -87,3 +87,29 @@ else
     <a asp-action="Add" asp-controller="Home" class="btn btn-outline-primary">Add a PR</a>
   }
 }
+<script type="text/javascript">
```
Comment (Layla-P) [inline HacktoberfestProject.Web/Views/Home/Index.cshtml:90]: We just need to wrap the `<script></script>` tags in a section :-)

```
@section Scripts {
<script type="text/javascript">
.....
.....
</script>
}
```

**Review by Layla-P — CHANGES_REQUESTED** [review]
Just a very minor change and then you're good to go! Thank you for contributing :-)

File: HacktoberfestProject.Web/Views/Home/Index.cshtml (hunk 1 above)
Comment (SekharV) [reply HacktoberfestProject.Web/Views/Home/Index.cshtml]: Done @Layla-P

**Review by Layla-P — APPROVED** [review]
Thank you so much for the contribution!

**Pull request merged**